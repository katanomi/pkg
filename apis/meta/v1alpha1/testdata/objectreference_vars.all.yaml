kind: "SomeKind"
namespace: "default"
name: "some-object"
uid: "41ce460c-cd91-493f-a59c-8f2ccdea9034"
apiVersion: somegroup/v1alpha1
