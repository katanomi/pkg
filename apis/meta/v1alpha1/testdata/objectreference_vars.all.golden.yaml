ref: ""
ref.kind: "SomeKind"
ref.namespace: "default"
ref.name: "some-object"
ref.uid: "41ce460c-cd91-493f-a59c-8f2ccdea9034"
ref.apiVersion: somegroup/v1alpha1
